### ISSUE TITLE
Pipenv doesn't install latest version of celery

### ISSUE THREAD

**Issue by mdczaplicki** [pypa/pipenv#3719]
### Issue description

Pipenv doesn't install latest version of celery. It installs 3.1.26.post2 instead of 4.3.0 which is the most recent.

$ pipenv install celery==4.3.0
Running $ pipenv lock then $ pipenv sync.
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
✘ Locking Failed! 
[pipenv.exceptions.ResolutionFailure]:       req_dir=requirements_dir
[pipenv.exceptions.ResolutionFailure]:   File "/home/marek/.local/lib/python3.6/site-packages/pipenv/utils.py", line 726, in resolve_deps
[pipenv.exceptions.ResolutionFailure]:       req_dir=req_dir,
[pipenv.exceptions.ResolutionFailure]:   File "/home/marek/.local/lib/python3.6/site-packages/pipenv/utils.py", line 480, in actually_resolve_deps
[pipenv.exceptions.ResolutionFailure]:       resolved_tree = resolver.resolve()
[pipenv.exceptions.ResolutionFailure]:   File "/home/marek/.local/lib/python3.6/site-packages/pipenv/utils.py", line 395, in resolve
[pipenv.exceptions.ResolutionFailure]:       raise ResolutionFailure(message=str(e))
[pipenv.exceptions.ResolutionFailure]:       pipenv.exceptions.ResolutionFailure: ERROR: ERROR: Could not find a version that matches celery<4.0,==4.3.0,>=3.1.15
[pipenv.exceptions.ResolutionFailure]:       Tried: 0.1.2, 0.1.4, 0.1.6, 0.1.7, 0.1.8, 0.1.10, 0.1.11, 0.1.12, 0.1.13, 0.1.14, 0.1.15, 0.2.0, 0.3.0, 0.3.7, 0.3.20, 0.4.0, 0.4.1, 0.6.0, 0.8.0, 0.8.1, 0.8.2, 0.8.3, 0.8.4, 1.0.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.1.0, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.2.0, 2.2.0, 2.2.0, 2.2.1, 2.2.1, 2.2.1, 2.2.2, 2.2.2, 2.2.2, 2.2.3, 2.2.3, 2.2.3, 2.2.4, 2.2.5, 2.2.6, 2.2.7, 2.2.8, 2.2.9, 2.2.10, 2.3.0, 2.3.1, 2.3.2, 2.3.3, 2.3.4, 2.3.5, 2.4.0, 2.4.1, 2.4.2, 2.4.3, 2.4.4, 2.4.5, 2.4.6, 2.4.7, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.5.5, 3.0.0, 3.0.1, 3.0.2, 3.0.3, 3.0.4, 3.0.5, 3.0.6, 3.0.7, 3.0.8, 3.0.9, 3.0.10, 3.0.11, 3.0.12, 3.0.13, 3.0.14, 3.0.15, 3.0.16, 3.0.17, 3.0.18, 3.0.19, 3.0.20, 3.0.21, 3.0.22, 3.0.23, 3.0.24, 3.0.25, 3.1.0, 3.1.1, 3.1.2, 3.1.3, 3.1.3, 3.1.3, 3.1.3, 3.1.4, 3.1.5, 3.1.6, 3.1.7, 3.1.8, 3.1.9, 3.1.9, 3.1.10, 3.1.10, 3.1.11, 3.1.11, 3.1.12, 3.1.12, 3.1.13, 3.1.13, 3.1.14, 3.1.14, 3.1.15, 3.1.15, 3.1.16, 3.1.16, 3.1.17, 3.1.17, 3.1.18, 3.1.18, 3.1.19, 3.1.19, 3.1.20, 3.1.20, 3.1.21, 3.1.21, 3.1.22, 3.1.22, 3.1.23, 3.1.23, 3.1.24, 3.1.24, 3.1.25, 3.1.25, 3.1.26.post1, 3.1.26.post1, 3.1.26.post2, 3.1.26.post2, 4.0.0, 4.0.0, 4.0.1, 4.0.1, 4.0.2, 4.0.2, 4.1.0, 4.1.0, 4.1.1, 4.1.1, 4.2.0, 4.2.0, 4.2.1, 4.2.1, 4.2.2, 4.2.2, 4.3.0, 4.3.0
[pipenv.exceptions.ResolutionFailure]:       Skipped pre-versions: 4.0.0rc3, 4.0.0rc3, 4.0.0rc4, 4.0.0rc4, 4.0.0rc5, 4.0.0rc5, 4.0.0rc6, 4.0.0rc6, 4.0.0rc7, 4.0.0rc7, 4.2.0rc1, 4.2.0rc1, 4.2.0rc2, 4.2.0rc2, 4.2.0rc3, 4.2.0rc3, 4.2.0rc4, 4.2.0rc4, 4.3.0rc1, 4.3.0rc1, 4.3.0rc2, 4.3.0rc2, 4.3.0rc3, 4.3.0rc3
[pipenv.exceptions.ResolutionFailure]: Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  First try clearing your dependency cache with $ pipenv lock --clear, then try the original command again.
 Alternatively, you can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
  Hint: try $ pipenv lock --pre if it is a pre-release dependency.
ERROR: ERROR: Could not find a version that matches celery<4.0,==4.3.0,>=3.1.15
Tried: 0.1.2, 0.1.4, 0.1.6, 0.1.7, 0.1.8, 0.1.10, 0.1.11, 0.1.12, 0.1.13, 0.1.14, 0.1.15, 0.2.0, 0.3.0, 0.3.7, 0.3.20, 0.4.0, 0.4.1, 0.6.0, 0.8.0, 0.8.1, 0.8.2, 0.8.3, 0.8.4, 1.0.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.1.0, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.2.0, 2.2.0, 2.2.0, 2.2.1, 2.2.1, 2.2.1, 2.2.2, 2.2.2, 2.2.2, 2.2.3, 2.2.3, 2.2.3, 2.2.4, 2.2.5, 2.2.6, 2.2.7, 2.2.8, 2.2.9, 2.2.10, 2.3.0, 2.3.1, 2.3.2, 2.3.3, 2.3.4, 2.3.5, 2.4.0, 2.4.1, 2.4.2, 2.4.3, 2.4.4, 2.4.5, 2.4.6, 2.4.7, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.5.5, 3.0.0, 3.0.1, 3.0.2, 3.0.3, 3.0.4, 3.0.5, 3.0.6, 3.0.7, 3.0.8, 3.0.9, 3.0.10, 3.0.11, 3.0.12, 3.0.13, 3.0.14, 3.0.15, 3.0.16, 3.0.17, 3.0.18, 3.0.19, 3.0.20, 3.0.21, 3.0.22, 3.0.23, 3.0.24, 3.0.25, 3.1.0, 3.1.1, 3.1.2, 3.1.3, 3.1.3, 3.1.3, 3.1.3, 3.1.4, 3.1.5, 3.1.6, 3.1.7, 3.1.8, 3.1.9, 3.1.9, 3.1.10, 3.1.10, 3.1.11, 3.1.11, 3.1.12, 3.1.12, 3.1.13, 3.1.13, 3.1.14, 3.1.14, 3.1.15, 3.1.15, 3.1.16, 3.1.16, 3.1.17, 3.1.17, 3.1.18, 3.1.18, 3.1.19, 3.1.19, 3.1.20, 3.1.20, 3.1.21, 3.1.21, 3.1.22, 3.1.22, 3.1.23, 3.1.23, 3.1.24, 3.1.24, 3.1.25, 3.1.25, 3.1.26.post1, 3.1.26.post1, 3.1.26.post2, 3.1.26.post2, 4.0.0, 4.0.0, 4.0.1, 4.0.1, 4.0.2, 4.0.2, 4.1.0, 4.1.0, 4.1.1, 4.1.1, 4.2.0, 4.2.0, 4.2.1, 4.2.1, 4.2.2, 4.2.2, 4.3.0, 4.3.0
Skipped pre-versions: 4.0.0rc3, 4.0.0rc3, 4.0.0rc4, 4.0.0rc4, 4.0.0rc5, 4.0.0rc5, 4.0.0rc6, 4.0.0rc6, 4.0.0rc7, 4.0.0rc7, 4.2.0rc1, 4.2.0rc1, 4.2.0rc2, 4.2.0rc2, 4.2.0rc3, 4.2.0rc3, 4.2.0rc4, 4.2.0rc4, 4.3.0rc1, 4.3.0rc1, 4.3.0rc2, 4.3.0rc2, 4.3.0rc3, 4.3.0rc3
There are incompatible versions in the resolved dependencies.
[pipenv.exceptions.ResolutionFailure]:   File "/home/marek/.local/lib/python3.6/site-packages/pipenv/utils.py", line 726, in resolve_deps
[pipenv.exceptions.ResolutionFailure]:       req_dir=req_dir,
[pipenv.exceptions.ResolutionFailure]:   File "/home/marek/.local/lib/python3.6/site-packages/pipenv/utils.py", line 480, in actually_resolve_deps
[pipenv.exceptions.ResolutionFailure]:       resolved_tree = resolver.resolve()
[pipenv.exceptions.ResolutionFailure]:   File "/home/marek/.local/lib/python3.6/site-packages/pipenv/utils.py", line 395, in resolve
[pipenv.exceptions.ResolutionFailure]:       raise ResolutionFailure(message=str(e))
[pipenv.exceptions.ResolutionFailure]:       pipenv.exceptions.ResolutionFailure: ERROR: ERROR: Could not find a version that matches celery<4.0,==4.3.0,>=3.1.15
[pipenv.exceptions.ResolutionFailure]:       Tried: 0.1.2, 0.1.4, 0.1.6, 0.1.7, 0.1.8, 0.1.10, 0.1.11, 0.1.12, 0.1.13, 0.1.14, 0.1.15, 0.2.0, 0.3.0, 0.3.7, 0.3.20, 0.4.0, 0.4.1, 0.6.0, 0.8.0, 0.8.1, 0.8.2, 0.8.3, 0.8.4, 1.0.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.1.0, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.2.0, 2.2.0, 2.2.0, 2.2.1, 2.2.1, 2.2.1, 2.2.2, 2.2.2, 2.2.2, 2.2.3, 2.2.3, 2.2.3, 2.2.4, 2.2.5, 2.2.6, 2.2.7, 2.2.8, 2.2.9, 2.2.10, 2.3.0, 2.3.1, 2.3.2, 2.3.3, 2.3.4, 2.3.5, 2.4.0, 2.4.1, 2.4.2, 2.4.3, 2.4.4, 2.4.5, 2.4.6, 2.4.7, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.5.5, 3.0.0, 3.0.1, 3.0.2, 3.0.3, 3.0.4, 3.0.5, 3.0.6, 3.0.7, 3.0.8, 3.0.9, 3.0.10, 3.0.11, 3.0.12, 3.0.13, 3.0.14, 3.0.15, 3.0.16, 3.0.17, 3.0.18, 3.0.19, 3.0.20, 3.0.21, 3.0.22, 3.0.23, 3.0.24, 3.0.25, 3.1.0, 3.1.1, 3.1.2, 3.1.3, 3.1.3, 3.1.3, 3.1.3, 3.1.4, 3.1.5, 3.1.6, 3.1.7, 3.1.8, 3.1.9, 3.1.9, 3.1.10, 3.1.10, 3.1.11, 3.1.11, 3.1.12, 3.1.12, 3.1.13, 3.1.13, 3.1.14, 3.1.14, 3.1.15, 3.1.15, 3.1.16, 3.1.16, 3.1.17, 3.1.17, 3.1.18, 3.1.18, 3.1.19, 3.1.19, 3.1.20, 3.1.20, 3.1.21, 3.1.21, 3.1.22, 3.1.22, 3.1.23, 3.1.23, 3.1.24, 3.1.24, 3.1.25, 3.1.25, 3.1.26.post1, 3.1.26.post1, 3.1.26.post2, 3.1.26.post2, 4.0.0, 4.0.0, 4.0.1, 4.0.1, 4.0.2, 4.0.2, 4.1.0, 4.1.0, 4.1.1, 4.1.1, 4.2.0, 4.2.0, 4.2.1, 4.2.1, 4.2.2, 4.2.2, 4.3.0, 4.3.0
[pipenv.exceptions.ResolutionFailure]:       Skipped pre-versions: 4.0.0rc3, 4.0.0rc3, 4.0.0rc4, 4.0.0rc4, 4.0.0rc5, 4.0.0rc5, 4.0.0rc6, 4.0.0rc6, 4.0.0rc7, 4.0.0rc7, 4.2.0rc1, 4.2.0rc1, 4.2.0rc2, 4.2.0rc2, 4.2.0rc3, 4.2.0rc3, 4.2.0rc4, 4.2.0rc4, 4.3.0rc1, 4.3.0rc1, 4.3.0rc2, 4.3.0rc2, 4.3.0rc3, 4.3.0rc3
[pipenv.exceptions.ResolutionFailure]: Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  First try clearing your dependency cache with $ pipenv lock --clear, then try the original command again.
 Alternatively, you can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
  Hint: try $ pipenv lock --pre if it is a pre-release dependency.
ERROR: ERROR: Could not find a version that matches celery<4.0,==4.3.0,>=3.1.15
Tried: 0.1.2, 0.1.4, 0.1.6, 0.1.7, 0.1.8, 0.1.10, 0.1.11, 0.1.12, 0.1.13, 0.1.14, 0.1.15, 0.2.0, 0.3.0, 0.3.7, 0.3.20, 0.4.0, 0.4.1, 0.6.0, 0.8.0, 0.8.1, 0.8.2, 0.8.3, 0.8.4, 1.0.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.1.0, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.2.0, 2.2.0, 2.2.0, 2.2.1, 2.2.1, 2.2.1, 2.2.2, 2.2.2, 2.2.2, 2.2.3, 2.2.3, 2.2.3, 2.2.4, 2.2.5, 2.2.6, 2.2.7, 2.2.8, 2.2.9, 2.2.10, 2.3.0, 2.3.1, 2.3.2, 2.3.3, 2.3.4, 2.3.5, 2.4.0, 2.4.1, 2.4.2, 2.4.3, 2.4.4, 2.4.5, 2.4.6, 2.4.7, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.5.5, 3.0.0, 3.0.1, 3.0.2, 3.0.3, 3.0.4, 3.0.5, 3.0.6, 3.0.7, 3.0.8, 3.0.9, 3.0.10, 3.0.11, 3.0.12, 3.0.13, 3.0.14, 3.0.15, 3.0.16, 3.0.17, 3.0.18, 3.0.19, 3.0.20, 3.0.21, 3.0.22, 3.0.23, 3.0.24, 3.0.25, 3.1.0, 3.1.1, 3.1.2, 3.1.3, 3.1.3, 3.1.3, 3.1.3, 3.1.4, 3.1.5, 3.1.6, 3.1.7, 3.1.8, 3.1.9, 3.1.9, 3.1.10, 3.1.10, 3.1.11, 3.1.11, 3.1.12, 3.1.12, 3.1.13, 3.1.13, 3.1.14, 3.1.14, 3.1.15, 3.1.15, 3.1.16, 3.1.16, 3.1.17, 3.1.17, 3.1.18, 3.1.18, 3.1.19, 3.1.19, 3.1.20, 3.1.20, 3.1.21, 3.1.21, 3.1.22, 3.1.22, 3.1.23, 3.1.23, 3.1.24, 3.1.24, 3.1.25, 3.1.25, 3.1.26.post1, 3.1.26.post1, 3.1.26.post2, 3.1.26.post2, 4.0.0, 4.0.0, 4.0.1, 4.0.1, 4.0.2, 4.0.2, 4.1.0, 4.1.0, 4.1.1, 4.1.1, 4.2.0, 4.2.0, 4.2.1, 4.2.1, 4.2.2, 4.2.2, 4.3.0, 4.3.0
Skipped pre-versions: 4.0.0rc3, 4.0.0rc3, 4.0.0rc4, 4.0.0rc4, 4.0.0rc5, 4.0.0rc5, 4.0.0rc6, 4.0.0rc6, 4.0.0rc7, 4.0.0rc7, 4.2.0rc1, 4.2.0rc1, 4.2.0rc2, 4.2.0rc2, 4.2.0rc3, 4.2.0rc3, 4.2.0rc4, 4.2.0rc4, 4.3.0rc1, 4.3.0rc1, 4.3.0rc2, 4.3.0rc2, 4.3.0rc3, 4.3.0rc3
There are incompatible versions in the resolved dependencies.

### Expected result

For pipenv to install celery 4.3.0

### Actual result

Pipenv installed celery 3.1.26.post2

### Steps to replicate

pipenv install celery
pipenv install celery==4.3.0

-------------------------------------------------------------------------------
<details><summary>$ pipenv --support</summary>

Pipenv version: `'2018.11.26'`

Pipenv location: `'/home/marek/.local/share/virtualenvs/in4data-N5yI1hIr/lib/python3.6/site-packages/pipenv'`

Python location: `'/home/marek/.local/share/virtualenvs/in4data-N5yI1hIr/bin/python3.6'`

Python installations found:

  - `3.7.3`: `/usr/local/bin/python3.7m`
  - `3.7.3`: `/usr/local/bin/python3.7`
  - `3.7.3`: `/usr/bin/python3.7m`
  - `3.7.3`: `/usr/bin/python3.7`
  - `3.6.7`: `/home/marek/.local/share/virtualenvs/in4data-N5yI1hIr/bin/python`
  - `3.6.7`: `/usr/bin/python3.6m`
  - `3.6.7`: `/usr/bin/python3.6`
  - `2.7.15rc1`: `/usr/bin/python2`

PEP 508 Information:

```
{'implementation_name': 'cpython',
 'implementation_version': '3.6.7',
 'os_name': 'posix',
 'platform_machine': 'x86_64',
 'platform_python_implementation': 'CPython',
 'platform_release': '4.15.0-47-generic',
 'platform_system': 'Linux',
 'platform_version': '#50-Ubuntu SMP Wed Mar 13 10:44:52 UTC 2019',
 'python_full_version': '3.6.7',
 'python_version': '3.6',
 'sys_platform': 'linux'}
```

System environment variables:

  - `LS_COLORS`
  - `TERMINAL_EMULATOR`
  - `LC_MEASUREMENT`
  - `LESSCLOSE`
  - `LC_PAPER`
  - `LC_MONETARY`
  - `SNAP_USER_DATA`
  - `LANG`
  - `GDM_LANG`
  - `DISPLAY`
  - `OLDPWD`
  - `SNAP_REVISION`
  - `GTK_OVERLAY_SCROLLING`
  - `SNAP_ARCH`
  - `SNAP_INSTANCE_KEY`
  - `XDG_VTNR`
  - `GIO_LAUNCHED_DESKTOP_FILE_PID`
  - `SSH_AUTH_SOCK`
  - `VIRTUAL_ENV`
  - `MANDATORY_PATH`
  - `LC_NAME`
  - `XDG_SESSION_ID`
  - `XDG_GREETER_DATA_DIR`
  - `SNAP_USER_COMMON`
  - `USER`
  - `DESKTOP_SESSION`
  - `PIP_PYTHON_PATH`
  - `DEFAULTS_PATH`
  - `QT_QPA_PLATFORMTHEME`
  - `PWD`
  - `HOME`
  - `SNAP`
  - `SSH_AGENT_PID`
  - `QT_ACCESSIBILITY`
  - `SNAP_COMMON`
  - `XDG_SESSION_TYPE`
  - `SNAP_NAME`
  - `XDG_DATA_DIRS`
  - `SNAP_INSTANCE_NAME`
  - `SNAP_DATA`
  - `PIPENV_ACTIVE`
  - `XDG_SESSION_DESKTOP`
  - `LC_ADDRESS`
  - `GJS_DEBUG_OUTPUT`
  - `LC_NUMERIC`
  - `GTK_MODULES`
  - `SNAP_COOKIE`
  - `SHELL`
  - `TERM`
  - `XDG_SEAT_PATH`
  - `XDG_CURRENT_DESKTOP`
  - `GPG_AGENT_INFO`
  - `BAMF_DESKTOP_FILE_HINT`
  - `SNAP_REEXEC`
  - `GIO_LAUNCHED_DESKTOP_FILE`
  - `PYTHONDONTWRITEBYTECODE`
  - `XDG_SEAT`
  - `SHLVL`
  - `LANGUAGE`
  - `LC_TELEPHONE`
  - `GDMSESSION`
  - `GNOME_DESKTOP_SESSION_ID`
  - `LOGNAME`
  - `PIP_DISABLE_PIP_VERSION_CHECK`
  - `DBUS_SESSION_BUS_ADDRESS`
  - `XDG_RUNTIME_DIR`
  - `XAUTHORITY`
  - `SNAP_CONTEXT`
  - `XDG_SESSION_PATH`
  - `XDG_CONFIG_DIRS`
  - `PATH`
  - `LC_IDENTIFICATION`
  - `PS1`
  - `CINNAMON_VERSION`
  - `SNAP_VERSION`
  - `GJS_DEBUG_TOPICS`
  - `SNAP_LIBRARY_PATH`
  - `SESSION_MANAGER`
  - `LESSOPEN`
  - `_`
  - `PIP_SHIMS_BASE_MODULE`
  - `PYTHONFINDER_IGNORE_UNSUPPORTED`

Pipenv–specific environment variables:

 - `PIPENV_ACTIVE`: `1`

Debug–specific environment variables:

  - `PATH`: `/home/marek/.local/share/virtualenvs/in4data-N5yI1hIr/bin:/home/marek/.local/share/virtualenvs/in4data-N5yI1hIr/bin:/home/marek/PycharmProjects/in4data/node_modules/.bin:/home/marek/.local/share/umake/bin:/home/marek/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin`
  - `SHELL`: `/bin/bash`
  - `LANG`: `en_US.UTF-8`
  - `PWD`: `/home/marek/PycharmProjects/in4data`
  - `VIRTUAL_ENV`: `/home/marek/.local/share/virtualenvs/in4data-N5yI1hIr`

---------------------------

Contents of `Pipfile` ('/home/marek/PycharmProjects/in4data/Pipfile'):

```toml
[[source]]
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true

[dev-packages]

[packages]
django = "*"
django-tables2 = "*"
django-crispy-forms = "*"
django-filter = "*"
django-compressor = "*"
libsass = "*"
django-libsass = "*"
mysqlclient = "*"
factory-boy = "*"
django-brutebuster2 = "*"
pillow = "*"
django-celery = "*"
celery = "==4.3.0"

[requires]
python_version = "3.6"

[pipenv]
allow_prereleases = true

```

Contents of `Pipfile.lock` ('/home/marek/PycharmProjects/in4data/Pipfile.lock'):

```json
{
    "_meta": {
        "hash": {
            "sha256": "39e7476c98449b2c43260052a97ac5247431063b5c228ca04e0a0d0785ed12c6"
        },
        "pipfile-spec": 6,
        "requires": {
            "python_version": "3.6"
        },
        "sources": [
            {
                "name": "pypi",
                "url": "https://pypi.org/simple",
                "verify_ssl": true
            }
        ]
    },
    "default": {
        "amqp": {
            "hashes": [
                "sha256:2dea4d16d073c902c3b89d9b96620fb6729ac0f7a923bbc777cb4ad827c0c61a",
                "sha256:e0ed0ce6b8ffe5690a2e856c7908dc557e0e605283d6885dd1361d79f2928908"
            ],
            "version": "==1.4.9"
        },
        "anyjson": {
            "hashes": [
                "sha256:37812d863c9ad3e35c0734c42e0bf0320ce8c3bed82cd20ad54cb34d158157ba"
            ],
            "version": "==0.3.3"
        },
        "billiard": {
            "hashes": [
                "sha256:204e75d390ef8f839c30a93b696bd842c3941916e15921745d05edc2a83868ab",
                "sha256:23cb71472712e96bff3e0d45763b7b8a99e5040385fffb96816028352c255682",
                "sha256:692a2a5a55ee39a42bcb7557930e2541da85df9ea81c6e24827f63b80cd39d0b",
                "sha256:82041dbaa62f7fde1464d7ab449978618a38b241b40c0d31dafabb36446635dc",
                "sha256:958fc9f8fd5cc9b936b2cb9d96f02aa5ec3613ba13ee7f089c77ff0bcc368fac",
                "sha256:c0cbe8d45ba8d8213ad68ef9a1881002a151569c9424d551634195a18c3a4160",
                "sha256:ccfe0419eb5e49f27ad35cf06e75360af903df6d576c66cb8073246d4e023e5c",
                "sha256:d4d2fed1a251ea58eed47b48db3778ebb92f5ff4407dc91869c6f41c3a9249d0"
            ],
            "version": "==3.3.0.23"
        },
        "celery": {
            "hashes": [
                "sha256:5493e172ae817b81ba7d09443ada114886765a8ce02f16a56e6fac68d953a9b2",
                "sha256:60211897aee321266ff043fe2b33eaac825dfe9f46843cf964fc97507a186334"
            ],
            "index": "pypi",
            "version": "==3.1.26.post2"
        },
        "django": {
            "hashes": [
                "sha256:7c3543e4fb070d14e10926189a7fcf42ba919263b7473dceaefce34d54e8a119",
                "sha256:a2814bffd1f007805b19194eb0b9a331933b82bd5da1c3ba3d7b7ba16e06dc4b"
            ],
            "index": "pypi",
            "version": "==2.2"
        },
        "django-appconf": {
            "hashes": [
                "sha256:35f13ca4d567f132b960e2cd4c832c2d03cb6543452d34e29b7ba10371ba80e3",
                "sha256:c98a7af40062e996b921f5962a1c4f3f0c979fa7885f7be4710cceb90ebe13a6"
            ],
            "version": "==1.0.3"
        },
        "django-brutebuster2": {
            "hashes": [
                "sha256:b4e65bad88c96b793300a075db51979851af9168a8c5b87ddcd29f0064dfedcb",
                "sha256:f5260925263f76fa21a6109617c7f2ac5c5e91189180be70adfb8dc72791724c"
            ],
            "index": "pypi",
            "version": "==0.2.0"
        },
        "django-celery": {
            "hashes": [
                "sha256:1450264db5ec58e45f3f20d8d361e696920352f62481bf56047288cdb38bcc0b",
                "sha256:aaba492bf7777f231ec6b02c80aa3ea68758c39f4723864dd4164589b99ad703"
            ],
            "index": "pypi",
            "version": "==3.2.2"
        },
        "django-compressor": {
            "hashes": [
                "sha256:7732676cfb9d58498dfb522b036f75f3f253f72ea1345ac036434fdc418c2e57",
                "sha256:9616570e5b08e92fa9eadc7a1b1b49639cce07ef392fc27c74230ab08075b30f"
            ],
            "index": "pypi",
            "version": "==2.2"
        },
        "django-crispy-forms": {
            "hashes": [
                "sha256:5952bab971110d0b86c278132dae0aa095beee8f723e625c3d3fa28888f1675f",
                "sha256:705ededc554ad8736157c666681165fe22ead2dec0d5446d65fc9dd976a5a876"
            ],
            "index": "pypi",
            "version": "==1.7.2"
        },
        "django-filter": {
            "hashes": [
                "sha256:3dafb7d2810790498895c22a1f31b2375795910680ac9c1432821cbedb1e176d",
                "sha256:a3014de317bef0cd43075a0f08dfa1d319a7ccc5733c3901fb860da70b0dda68"
            ],
            "index": "pypi",
            "version": "==2.1.0"
        },
        "django-libsass": {
            "hashes": [
                "sha256:49db3334b87e1f7955c4f9fb9945bc296f8bfd27a14d6d89706e4b0e5dc5de1c"
            ],
            "index": "pypi",
            "version": "==0.7"
        },
        "django-tables2": {
            "hashes": [
                "sha256:11d914910745d6826ba2226f45ee7fc38dbc471dd62992b4204c6ee68ec10689",
                "sha256:98038606603a157b79bb7564e589d1f21050c463d12c59d5b3848eba9e10a0ae"
            ],
            "index": "pypi",
            "version": "==2.0.6"
        },
        "factory-boy": {
            "hashes": [
                "sha256:6f25cc4761ac109efd503f096e2ad99421b1159f01a29dbb917359dcd68e08ca",
                "sha256:d552cb872b310ae78bd7429bf318e42e1e903b1a109e899a523293dfa762ea4f"
            ],
            "index": "pypi",
            "version": "==2.11.1"
        },
        "faker": {
            "hashes": [
                "sha256:167cef2454482dc2fbd8b0ff6a5ba3dbac8d3a3ebdee6ba819d008100d9d9428",
                "sha256:3f2f4570df28df2eb8f39b00520eb610081d6552975e926c6a2cbc64fd89c4c1"
            ],
            "version": "==1.0.5"
        },
        "kombu": {
            "hashes": [
                "sha256:7ceab743e3e974f3e5736082e8cc514c009e254e646d6167342e0e192aee81a6",
                "sha256:e064a00c66b4d1058cd2b0523fb8d98c82c18450244177b6c0f7913016642650"
            ],
            "version": "==3.0.37"
        },
        "libsass": {
            "hashes": [
                "sha256:2ae3b061a7d250fb47e5fdad1a8191600ca15dc604e76b109b6d3bf8e08fd2ed",
                "sha256:2ee186aa682a035a53c557b7e61ce562a1114f1a1a992d0ba962cbc3e82c490c",
                "sha256:366f4fd5a5eab4a519beb583e9fa78718cf2c0f40e92ed835d7ed23b82e5d954",
                "sha256:5511b3c62e8d97daf929c63bd516b794f0a06acd09dd261445d864e48290551b",
                "sha256:7462da168c8fb997b31cb4dc3ee5adb9af2d106f7b92c2d57a1c68a56ae5a3a0",
                "sha256:84a16ec5cf7842ff5bc2caed2c032ed624d587699797bc2a4d4a8e41f579b6e7",
                "sha256:8fc0360ee99224f7a3cb09987e641171d34180759f467ba3d15934102ade396f",
                "sha256:a6c5535a21a07d769151453270bc6a8373b821d1d2fd9810d84fccfe315ab188",
                "sha256:b375bfbf3c86ec0f4a27f266b44b2753a4b8cab7e73649eed7afcad84bc56257",
                "sha256:b548af46c1a606aed93da2566901146005d6065f73fefc63d256ba62ba1f803d",
                "sha256:bb30fc7125350c64925a98cb90da7979f76bb0ea1a0157e8aeb268f8da38e296",
                "sha256:c2f386677514f9fc758631328bd318dd3e9d839ad7b6e248ec4535a191bfd271",
                "sha256:d1f301637ad5768aecc81d17dcf40a68f2e11b7ca8b427dbb9f8972c150d303e",
                "sha256:e0cf54dddf2cc6e373005bed6e46ccdce1f3a77bd169ab505c3a8ad9023eee5f",
                "sha256:e8941881063691d50f9cc8b8d6d8fd7bec86a8c461b2a4fc87188a5fc44d6ba4",
                "sha256:f4b29b0c70d753c754a58aaad7c31ad3309ca4a26f9aa64e695157251f6832ad"
            ],
            "index": "pypi",
            "version": "==0.18.0"
        },
        "mysqlclient": {
            "hashes": [
                "sha256:425e733b05e359a714d6007c0fc44582be66b63e5a3df0a50949274ae16f4bc6",
                "sha256:62e4770b6a797b9416bcf70488365b7d6b9c9066878108499c559293bb464380",
                "sha256:f257d250f2675d0ef99bd318906f3cfc05cef4a2f385ea695ff32a3f04b9f9a7"
            ],
            "index": "pypi",
            "version": "==1.4.2.post1"
        },
        "pillow": {
            "hashes": [
                "sha256:15c056bfa284c30a7f265a41ac4cbbc93bdbfc0dfe0613b9cb8a8581b51a9e55",
                "sha256:1a4e06ba4f74494ea0c58c24de2bb752818e9d504474ec95b0aa94f6b0a7e479",
                "sha256:1c3c707c76be43c9e99cb7e3d5f1bee1c8e5be8b8a2a5eeee665efbf8ddde91a",
                "sha256:1fd0b290203e3b0882d9605d807b03c0f47e3440f97824586c173eca0aadd99d",
                "sha256:24114e4a6e1870c5a24b1da8f60d0ba77a0b4027907860188ea82bd3508c80eb",
                "sha256:258d886a49b6b058cd7abb0ab4b2b85ce78669a857398e83e8b8e28b317b5abb",
                "sha256:33c79b6dd6bc7f65079ab9ca5bebffb5f5d1141c689c9c6a7855776d1b09b7e8",
                "sha256:367385fc797b2c31564c427430c7a8630db1a00bd040555dfc1d5c52e39fcd72",
                "sha256:3c1884ff078fb8bf5f63d7d86921838b82ed4a7d0c027add773c2f38b3168754",
                "sha256:44e5240e8f4f8861d748f2a58b3f04daadab5e22bfec896bf5434745f788f33f",
                "sha256:46aa988e15f3ea72dddd81afe3839437b755fffddb5e173886f11460be909dce",
                "sha256:74d90d499c9c736d52dd6d9b7221af5665b9c04f1767e35f5dd8694324bd4601",
                "sha256:809c0a2ce9032cbcd7b5313f71af4bdc5c8c771cb86eb7559afd954cab82ebb5",
                "sha256:85d1ef2cdafd5507c4221d201aaf62fc9276f8b0f71bd3933363e62a33abc734",
                "sha256:8c3889c7681af77ecfa4431cd42a2885d093ecb811e81fbe5e203abc07e0995b",
                "sha256:9218d81b9fca98d2c47d35d688a0cea0c42fd473159dfd5612dcb0483c63e40b",
                "sha256:9aa4f3827992288edd37c9df345783a69ef58bd20cc02e64b36e44bcd157bbf1",
                "sha256:9d80f44137a70b6f84c750d11019a3419f409c944526a95219bea0ac31f4dd91",
                "sha256:b7ebd36128a2fe93991293f997e44be9286503c7530ace6a55b938b20be288d8",
                "sha256:c4c78e2c71c257c136cdd43869fd3d5e34fc2162dc22e4a5406b0ebe86958239",
                "sha256:c6a842537f887be1fe115d8abb5daa9bc8cc124e455ff995830cc785624a97af",
                "sha256:cf0a2e040fdf5a6d95f4c286c6ef1df6b36c218b528c8a9158ec2452a804b9b8",
                "sha256:cfd28aad6fc61f7a5d4ee556a997dc6e5555d9381d1390c00ecaf984d57e4232",
                "sha256:dca5660e25932771460d4688ccbb515677caaf8595f3f3240ec16c117deff89a",
                "sha256:de7aedc85918c2f887886442e50f52c1b93545606317956d65f342bd81cb4fc3",
                "sha256:e6c0bbf8e277b74196e3140c35f9a1ae3eafd818f7f2d3a15819c49135d6c062"
            ],
            "index": "pypi",
            "version": "==6.0.0"
        },
        "python-dateutil": {
            "hashes": [
                "sha256:7e6584c74aeed623791615e26efd690f29817a27c73085b78e4bad02493df2fb",
                "sha256:c89805f6f4d64db21ed966fda138f8a5ed7a4fdbc1a8ee329ce1b74e3c74da9e"
            ],
            "version": "==2.8.0"
        },
        "pytz": {
            "hashes": [
                "sha256:303879e36b721603cc54604edcac9d20401bdbe31e1e4fdee5b9f98d5d31dfda",
                "sha256:d747dd3d23d77ef44c6a3526e274af6efeb0a6f1afd5a69ba4d5be4098c8e141"
            ],
            "version": "==2019.1"
        },
        "rcssmin": {
            "hashes": [
                "sha256:ca87b695d3d7864157773a61263e5abb96006e9ff0e021eff90cbe0e1ba18270"
            ],
            "version": "==1.0.6"
        },
        "rjsmin": {
            "hashes": [
                "sha256:dd9591aa73500b08b7db24367f8d32c6470021f39d5ab4e50c7c02e4401386f1"
            ],
            "version": "==1.0.12"
        },
        "six": {
            "hashes": [
                "sha256:3350809f0555b11f552448330d0b52d5f24c91a322ea4a15ef22629740f3761c",
                "sha256:d16a0141ec1a18405cd4ce8b4613101da75da0e9a7aec5bdd4fa804d0e0eba73"
            ],
            "version": "==1.12.0"
        },
        "sqlparse": {
            "hashes": [
                "sha256:40afe6b8d4b1117e7dff5504d7a8ce07d9a1b15aeeade8a2d10f130a834f8177",
                "sha256:7c3dca29c022744e95b547e867cee89f4fce4373f3549ccd8797d8eb52cdb873"
            ],
            "version": "==0.3.0"
        },
        "text-unidecode": {
            "hashes": [
                "sha256:5a1375bb2ba7968740508ae38d92e1f889a0832913cb1c447d5e2046061a396d",
                "sha256:801e38bd550b943563660a91de8d4b6fa5df60a542be9093f7abf819f86050cc"
            ],
            "version": "==1.2"
        }
    },
    "develop": {}
}

```
</details>

**Comment by gsemet** [comment]
Look like duplicate of #3568

**Comment by gsemet** [comment]
"passa" seems the mid-term solution :) Hopefully it should solve such issue once it lands.

**Comment by mdczaplicki** [comment]
I seem not to find any workaround in this thread, did you come up with any?

**Comment by jxltom** [comment]
@mdczaplicki You could try https://passa.readthedocs.io/en/latest/ until passa is ported into pipenv

**Comment by techalchemy** [comment]
Pipenv has no actual trouble installing celery if you install it alone. `django-celery` seems to require `celery<4.0` however, which you can validate by running `pipenv lock --verbose` and seeing the resolution conflict:

```console
  django-celery==3.3.0      requires celery<4.0,>=3.1.15, django>=1.8
```

As a result there will naturally always be a conflict if you try to strictly pin a higher version of celery. You have a legitimate conflict in your dependency graph it appears. Sorry for your confusion, you should probably just let the resolver determine the version of celery to install for you

**Comment by nour-sidaoui** [comment]
uninstall celery and install django-celery by itself (it'll include the matching celery version)
Hope it helps :)
